tailwind.config.js

```diff
@@ -20,7 +21,10 @@ module.exports = {
       colors: {
         primary: "#165C9C",
         secondary: "#101828",
-        gray: "#667085"
+        gray: "#667085",
+        dark: "#000000",
+        altDark: "#222222",
+        altWhite: "#F0F0F0"
```

Comment:
Could you also elaborate on why you put the colors here as well please?

Reply:
Just Incase we want to change the color generally we could just change it from the tailwind config